### DISCUSSION TITLE
Add Link for Helm deploy?

### DISCUSSION THREAD

**Discussion by rr-guyle-taber** [amundsen-io/amundsen#1418] [Q&A]
Does the Amundsen helm repo have a publicly available add link URI I can use in my helm deploy?

Something similar to what Argo is doing? `helm repo add argo https://argoproj.github.io/argo-helm`

**Comment by rr-guyle-taber** [comment]
> Does the Amundsen helm repo have a publicly available add link URI I can use in my helm deploy?
> 
> Something similar to what Argo is doing? `helm repo add argo https://argoproj.github.io/argo-helm`

```
helm repo add amundsen https://github.com/amundsen-io/amundsen/tree/main/amundsen-kube-helm/templates/helm

Error: looks like "https://github.com/amundsen-io/amundsen/tree/main/amundsen-kube-helm/templates/helm" is not a valid chart repository or cannot be reached: failed to fetch https://github.com/amundsen-io/amundsen/tree/main/amundsen-kube-helm/templates/helm/index.yaml : 404 Not Found

```

**Reply by lhoss** [reply]
I was wondering the same !
Found there's an open issue about the same question https://github.com/amundsen-io/amundsen/issues/382
( Not sure yet now what's the easiest way to get that helm chart installed on OpenShift )
